src/psyclone/psyGen.py

```diff
@@ -556,13 +556,18 @@ def __init__(self, alg_invocation, idx, schedule_class, invokes,
         if not reserved_names:
             reserved_names = []
 
+        # Get a reference to the parent container, if any
+        container = None
```

Comment:
Done. Was hard to test in isolation because constructing an `Invokes` object requires a PSy object but constructing that also constructs an `Invokes` object.